.github/workflows/ci.yml

```diff
@@ -26,7 +26,7 @@ jobs:
           node-version: ${{ matrix.node-version }}
 
       - name: Cache node modules
-        uses: actions/cache@v1
+        uses: actions/cache@v4
```

Comment:
To fix the v1 deprecation issue

![Screenshot 2025-03-13 at 10 18 06](https://github.com/user-attachments/assets/df2035d8-2f80-4d33-bd8e-a275001f250c)